include/ivmgrHeadless.h

```diff
@@ -0,0 +1,150 @@
+//######################################################################
+//
+// GraspIt!
+// Copyright (C) 2002-2009  Columbia University in the City of New York.
+// All rights reserved.
+//
+// GraspIt! is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// GraspIt! is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with GraspIt!.  If not, see <http://www.gnu.org/licenses/>.
+//
+// Author(s):  Jennifer Buehler 
+//
+//######################################################################
+
+/*! \file 
+  \brief Defines the IVmgrHeadless class which acts like the original IVmgrHeadless (handling 3D user interaction), but bypasses all GUI related actions.
+ */
+#ifndef IVMGR_NOGUI_HXX
+#include <Inventor/SbBasic.h>
+#include <Inventor/SbLinear.h>
+#include <QWidget>
+#include <list>
+#include <vector>
+#include <qstring.h>
+//#include <qwidget.h>
+#include "material.h"
+#include "ivmgrBase.h"
+
+//#include <Inventor/Qt/viewers/SoQtExaminerViewer.h>
+
+struct VCReportType;
+class SbVec3f;
+class transf;
+class position;
+
+class QWidget;
+class SoQtExaminerViewer;
+class SoBlinker;
+class SoCenterballDragger;
+class SoHandleBoxDragger;
+class SoTranslate1Dragger;
+class SoRotateDiscDragger;
+class SoDragger;
+class SoMaterial;
+class SoPath;
+class SoPickedPoint;
+class SoScale;
+class SoSelection;
+class SoSeparator;
+class SoText2;
+class SoTransform;
+class SoEventCallback;
+class SoSensor;
+class SoNodeSensor;
+class SoQtRenderArea;
+class World;
+class WorldElement;
+class Robot;
+class HumanHand;
+class KinematicChain;
+class DOF;
+class GraspableBody;
+class Body;
+class Finger;
+class Grasp;
+class GWS;
+class GWSprojection;
+class QualityMeasure;
+struct DraggerInfo;
+
```

Comment:
Again, well spotted. I forgot to tidy this up (my initial version was partly a copy of ivmgr). I'll take the unnecessary ones out.